src/main/java/uk/gov/justice/hmpps/prison/api/resource/LocationResource.java

```diff
@@ -72,7 +68,7 @@ public ResponseEntity<List<OffenderBooking>> getOffendersAtLocationDescription(
             @RequestHeader(value = "Sort-Fields", defaultValue = "lastName,firstName,bookingId", required = false) @Parameter(description = "Comma separated list of one or more of the following fields - <b>bookingNo, bookingId, offenderNo, firstName, lastName, agencyId, or assignedLivingUnitId</b>") final String sortFields,
             @RequestHeader(value = "Sort-Order", defaultValue = "ASC", required = false) @Parameter(description = "Sort order (ASC or DESC) - defaults to ASC.") final Order sortOrder) {
         final var request = SearchOffenderRequest.builder()
-                .username(authenticationFacade.getCurrentPrincipal())
+                .username(hmppsAuthenticationHolder.getPrincipal())
```

Comment:
couldn't work out whether we could get away with `username` here, so left it as `principal`